### ISSUE TITLE
Dynamically disable haptic handlers if user does not have haptic device

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-server#260]
Hello-haptics handler currently always creates haptic renderings. This will be confusing for users who don't have any haptic devices connected. Shared-Reality-Lab/auditory-haptic-graphics-browser#110 covers the implementation on the browser side for sending the status of Haply and Dot device presence for each request. This work item is for the handler(s) to pay attention to this and only create haptic renderings for device(s) that are actually present.

Assigning to @dreamseed87 to be given to the appropriate developer.

**Comment by jeffbl** [comment]
@dreamseed87 Are you doing this yourself? If not, can you please assign appropriately?

**Comment by rayanisran** [comment]
The hello-haptics handler has a check for whether a Haply device exists from the browser side, so my understanding is that a setting (checkbox?) needs to be added that can toggle this flag.

**Comment by dreamseed87** [comment]
Sorry I lost track this issue... We just talked, and in the handler, we have hasHaply() function. 
(Currently it never returns false though.)
We can read the checkbox value from the setting page, and it should be a few lines of flag variable handling. I assigned 
@sriGanna, @rayanisran for browser's haptics side, and @Clarisa1999 for UI (settings page) side.

**Comment by sriGanna** [comment]
For this issue, we are currently gated by Browser [#110](https://github.com/orgs/Shared-Reality-Lab/projects/1#card-75749085) which needs to progress enough to have a haptics option on the settings page.

**Comment by jeffbl** [comment]
My hope is that @Clarisa1999 and @sriGanna @rayanisran can agree on the schema, then both can proceed simultaneously? If adding this is complicated, suggest attending @JRegimbal office hours to hammer it out so this is not a bottleneck.

**Comment by jeffbl** [comment]
@rayanisran sorry, I just saw your comment. I'd sync up with @Clarisa1999 if you already implemented this, so that it can be migrated to the overall settings page?

**Comment by sriGanna** [comment]
Can't link the PR but this corresponds to PR#158 in the browser repo. It appears the PR is currently waiting to be reviewed

**Comment by jeffbl** [comment]
No, this is unrelated to PR#158...that is a separate issue. @Clarisa1999 indicates that the flag is already in and active, so it should just be a matter of the haptics handler paying attention to it.

**Comment by sriGanna** [comment]
Tested against unicorn using the nightly build. Should still be tested on pegasus when the haptics options page is fully deployed.
